### PR TITLE
Development : update "_cpp_parameters" and "CastroParticles.cpp"

### PR DESCRIPTION
**Pull request by taehoryu**
update "_cpp_parameters" and "CastroParticles.cpp"

### CODE REVIEW

**Comment by maxpkatz** [comment]
Can you please run Source/driver/mk_params.sh

File: Source/particles/CastroParticles.cpp [hunk 1]
```diff
@@ -236,20 +200,17 @@ Castro::TimestampParticles (int ngrow)
     {
 	first = false;
 
-	ParmParse ppp("particles");
 
 	// have to do it here, not in read_particle_params, because Density, ..., are set after
 	// read_particle_params is called.
 
 	int timestamp_density = 1;
```
Comment (zingale) [inline Source/particles/CastroParticles.cpp]: we also need to remove these default value settings -- these will now be done elsewhere in the code

File: Source/particles/CastroParticles.cpp [hunk 2]
```diff
@@ -236,20 +200,17 @@ Castro::TimestampParticles (int ngrow)
...
-	ppp.query("timestamp_density", timestamp_density);
 	if (timestamp_density) {
 	    timestamp_indices.push_back(Density);
 	    std::cout << "Density = " << Density << std::endl;
 	}
 
 	int timestamp_temperature = 0;
```
Comment (zingale) [inline Source/particles/CastroParticles.cpp]: need to remove this default

File: Source/driver/_cpp_parameters [hunk 3]
```diff
@@ -525,6 +525,45 @@ reset_checkpoint_step        int           -1
...
+
+#-----------------------------------------------------------------------------
+# category: particles
+#-----------------------------------------------------------------------------
+
+# permits tracer particle calculation to be turned on and off
+do_tracer_particles          int           0
+
+# whether the local densities at given positions of particles are stored in output files
+timestamp_density            int           0
```
Comment (zingale) [inline Source/driver/_cpp_parameters]: it looks like the old default value of this was `1` -- we should keep the defaults unchanged

**Comment by zingale** [comment]
I added some comments in the source

you need to also remove the default value of `particle_verbose` in `CastroParticles.cpp`

For `v`, we should declare it such that the query uses `particle_verbose` as the cpp_name, like:
`(v, verbose)`

(search for `verbose` in the `_cpp_parameters` file to see an example

File: Source/particles/CastroParticles.cpp [hunk 4]
```diff
@@ -11,7 +11,6 @@ using namespace amrex;
 
 AmrTracerParticleContainer* Castro::TracerPC =  0;
 int Castro::do_tracer_particles              =  0;
```
Comment (zingale) [inline Source/particles/CastroParticles.cpp]: this should also be removed

**Comment by zingale** [comment]
there's still some issues here.  I'm going to try to fix them on your fork via a PR that would then update this PR...

**Comment by zingale** [comment]
okay, I confirmed that the parameters do what they are supposed to do with this change

**Pull request merged**